### PR TITLE
Bump FlytePropeller version to populate FlyteWorkflow CRD labels

### PR DESCRIPTION
**Pull request by hamersaw**
# TL;DR
FlytePropeller sharding functionality relies on filtering FlyteWorkflow CRD labels for deterministic evaluation. [This PR](https://github.com/flyteorg/flytepropeller/pull/357/files) update the BuildFlyteWorkflow function within FlytePropeller to populate the required labels (i.e., shard-key, project, and domain). We need to propagate these changes through to FlyteAdmin by updating the FlytePropeller version.

## Type
 - [ ] Bug Fix
 - [X] Feature
 - [ ] Plugin

## Are all requirements met?

 - [X] Code completed
 - [X] Smoke tested
 - [X] Unit tests added
 - [X] Code documentation added
 - [X] Any pending items have an associated Issue

## Complete description
^^^

## Tracking Issue
_NA_

## Follow-up issue
_NA_

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ COPY . /go/src/github.com/flyteorg/flyteadmin/
 
 # This 'linux_compile' target should compile binaries to the /artifacts directory
 # The main entrypoint should be compiled to /artifacts/flyteadmin
-RUN make linux_compile
+RUN go mod tidy; make linux_compile
```
Comment (EngHabu) [inline Dockerfile:21]: Do we need this change?

Comment (hamersaw) [reply Dockerfile]: Thanks for catching this. I had made this change locally, never meant to check it in. Removed.

**Review by EngHabu — APPROVED** [review]
